INSTALL

```diff
@@ -19,11 +25,9 @@ It also contains different para-virtualized (PV) backends running inside it in o
 3. Machine dependent simple guest domain named "DomU". It has different PV frontends running and
 doesn't have any HW assigned except GPU. The latter is shared between "DomD" and "DomU" in current setup.
 
-It worth to mention that IPMMU for GPU is enabled in case of H3 ES3.0 only. For other supported SoCs the PVRKM
+It worth to mention that IPMMU for GPU is enabled in case of H3 ES3.0 and M3N only. For other supported SoCs the PVRKM
```

Comment:
while at it, could you please also s/It worth/it is worth ?

Reply:
Yes, sure